### PR TITLE
CLF Ship Nightmare Changes

### PR DESCRIPTION
**Pull request by toolmind**
# About the pull request

updates the CLF ship's airlocks to new generic ones and adds one CLF medic and one CLF engineer. (also a minor adjustment to the text about sentries not having IFF (they do (the intro text doesnt even appear for any survivors why has nobody re-added it))
(batu khan update coming soon)

# Explain why it's good for the game

The CLF ship should be very difficult to survive in, but not having anyone that can make folding cades/barricades quickly means the best option is to entirely ignore the cool new area of the map with potential for a cool holdout/last stand, and instead immediately abandon it to rush for a gap with the van. Having a medic also allows for the CLF survivors to have a chance at being revived instead of one death immediately putting them out of the game for good or meaning they'll spend the rest of the round in the permabrig if they get revived by the marines. I know a synth can spawn with them, but it's not a given at all that a person is going to be rolling synthvivor every round that the insert spawns in.

The doors were also outdated and took like 5 seconds to fully open, annoyance for survivors, xenos and marines alike.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: IowaPotatoFarmer
maptweak: Switched the outdated CLF ship airlocks to new generic ones.
add: The CLF crashed ship insert on LV-624 now has one guaranteed medic and one guaranteed engineer.
fix: The CLF crash intro text no longer includes inaccurate info about the sentries.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 1]
```diff
@@ -37,6 +37,32 @@
 	roundstart_damage_max = 10
 	roundstart_damage_times = 2
 
+	spawn_priority = SPAWN_PRIORITY_HIGH
+
+/obj/effect/landmark/survivor_spawner/lv624_crashed_clf_engineer
+	equipment = /datum/equipment_preset/clf/engineer
+	synth_equipment = /datum/equipment_preset/clf/synth
+	intro_text = list("<h2>You are a survivor of a crash landing!</h2>",\
+	"<span class='notice'>You are NOT aware of the xenomorph threat.</span>",\
```
Comment (harryob) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:46]:
```suggestion
	SPAN_NOTICE("You are NOT aware of the xenomorph threat."),\
```

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 2]
```diff
@@ -37,6 +37,32 @@
...
+	"<span class='danger'>Your primary objective is to heal up and survive. If you want to assault the hive - adminhelp.</span>")
```
Comment (harryob) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:47]:
```suggestion
	SPAN_DANGER("Your primary objective is to heal up and survive. If you want to assault the hive - adminhelp."))
```

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 3]
```diff
@@ -37,6 +37,32 @@
...
+	story_text = "You are a soldier of Colonial Liberation Front. Your ship received a distress signal from a planet bordering the CLF controlled space under USCM control. Ready and willing to save poor colonists from parasitic tyrants, you and your team boarded small ship called Marie Curie. Unfortunately, right before you came close to a landing zone, a glob of acid hit the ship, damaging one of the engines. Despite all the efforts of the pilot, the ship went straight into nearby mountain. You were hurt pretty badly in the crash. Dumbfounded, you rose up and noticed that one of your limbs is at a weird angle, broken. You looked at other survivors, also limping and trying to fix their broken bones. \red You also notice that sentry turrets on the ramp of your ship beep angrily, it seems their logic circuits got damaged. They will identify you as a foe, but at least they will identify everything as a foe."
```
Comment (harryob) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:48]:
```suggestion
	story_text = "You are a soldier of Colonial Liberation Front. Your ship received a distress signal from a planet bordering the CLF controlled space under USCM control. Ready and willing to save poor colonists from parasitic tyrants, you and your team boarded small ship called Marie Curie. Unfortunately, right before you came close to a landing zone, a glob of acid hit the ship, damaging one of the engines. Despite all the efforts of the pilot, the ship went straight into nearby mountain. You were hurt pretty badly in the crash. Dumbfounded, you rose up and noticed that one of your limbs is at a weird angle, broken. You looked at other survivors, also limping and trying to fix their broken bones. [SPAN_RED("You also notice that sentry turrets on the ramp of your ship beep angrily, it seems their logic circuits got damaged. They will identify you as a foe, but at least they will identify everything as a foe.")]"
```

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 4]
```diff
@@ -37,6 +37,32 @@
...
```
Comment (harryob) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:61]:
```suggestion
	SPAN_NOTICE("You are NOT aware of the xenomorph threat."),\
	SPAN_DANGER("Your primary objective is to heal up and survive. If you want to assault the hive - adminhelp."))
	story_text = "You are a soldier of Colonial Liberation Front. Your ship received a distress signal from a planet bordering the CLF controlled space under USCM control. Ready and willing to save poor colonists from parasitic tyrants, you and your team boarded small ship called Marie Curie. Unfortunately, right before you came close to a landing zone, a glob of acid hit the ship, damaging one of the engines. Despite all the efforts of the pilot, the ship went straight into nearby mountain. You were hurt pretty badly in the crash. Dumbfounded, you rose up and noticed that one of your limbs is at a weird angle, broken. You looked at other survivors, also limping and trying to fix their broken bones. [SPAN_RED("You also notice that sentry turrets on the ramp of your ship beep angrily, it seems their logic circuits got damaged. They will identify you as a foe, but at least they will identify everything as a foe.")]"
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Comment by toolmind** [comment]
harryOB GRIEFS my pr and cause check errors (evil). i'll see if i can fix it, probably going to remove that red spanned part about the sentries because they have CLF IFF now anyways.

**Comment by cm13live** [comment]
This pull request has been mentioned on **CM-SS13**. Please provide feedback here:

https://forum.cm-ss13.com/t/nightmare-survivor-spawn-rework/1795/3

**Review by harryob — APPROVED** [review]

**Pull request merged**